### ISSUE TITLE
Port changes to group and shared cloning acknowledgements

### ISSUE THREAD

**Issue by bree-z** [cuny-academic-commons/commons-in-a-box#351]
From: http://redmine.citytech.cuny.edu/issues/2934 

This would allow users to add acknowledgements to a group profile, and would also change the source for cloning acknowledgements data.

**Comment by boonebgorges** [comment]
I'll begin work on this after the OL team has signed off on the functionality as it appears on the OpenLab.

Notes to myself:
- We will need an `Upgrade` routine to Creators for existing groups
- I think we can probably manage to avoid adding new strings by stripping references to the group type in the user-facing text, but this will need testing from the OL team

**Comment by bree-z** [comment]
This has been tested with #354 , where I went through the matrix of different acknowledgements possibilities. 

The only outstanding issue is perhaps similar to http://redmine.citytech.cuny.edu/issues/2946 on the OpenLab.

When entering a non-member in the field, Enter doesn't act like an "I'm done" button. Instead, here's what happened:

- Testing in this group: https://openlabdev.commonsinabox.org/groups/course-with-additional-creator/ 
- Creators are (1) breefaculty (2) testfaculty
- I added 'Smoky the Bear' as a non-member and pressed Enter.
- The page reloaded and when I scrolled down Smoky the Bear was not added (see gif)
- Also, if I edit an existing non-member and then press enter, the non-member is deleted

![EnterDoesNotSaveNonMember](https://user-images.githubusercontent.com/7189824/127240795-ae5771b3-7e36-4f79-b19c-e52738da4a69.gif)
